src/Umbraco.Core/StringExtensions.cs

```diff
@@ -1478,5 +1478,31 @@ public static string ToSafeFileName(this string text, IShortStringHelper shortSt
         {
             return shortStringHelper.CleanStringForSafeFileName(text, culture);
         }
+
+        /// <summary>
+        /// Validates a string matches a time.
+        /// </summary>
+        /// <param name="input">Time representation (in H:mm format, e.g. 3:30, 22:00)</param>
+        /// <returns>True if provided input is a valid time in H:mm format, false if not.</returns>
+        public static bool IsValidTimeFormat(this string input)
+        {
+            if (string.IsNullOrEmpty(input) || !input.Contains(":") || input.Length < 4 || input.Length > 5)
```

Comment:
I still think this should be simply `Timespan.TryParse()`. Are we assuming invariant culture?

Reply:
Yes, you're right - I'll revert this.  I changed it on reflection thinking I'm trying to validate an actual time here, not a timestamp.  It's to validate a configured string is correct, which has previously been intended to be a time (i.e. hours and minutes only).  But now you've nudged me, there's no reason why this couldn't be a time more than 24 hours, so a TimeSpan is perfectly valid.